frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx

```diff
@@ -480,80 +481,28 @@ const User: React.FC<{
 	const { isLoggedIn } = useAuthContext()
 	const [truncated, setTruncated] = useState(true)
 
-	if (!errorObject?.session) {
-		return (
-			<Box width="full">
-				<Box pb="20" mt="12">
-					<Text weight="bold" size="large">
-						User details
-					</Text>
-				</Box>
-				<Callout title="We didn't find a session for this error">
-					<Box>
-						<Text size="small" weight="medium" color="moderate">
```

Comment:
I cleaned up some of the the duped JSX into `ErrorSessionMissingOrExcluded`. I confirmed it still looks the same as before:

![Screenshot 2023-05-11 at 2 07 51 PM](https://github.com/highlight/highlight/assets/58678/0943e618-07fc-4882-93af-cd8faf811c7e)